account_invoice_ubl_peppol/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+In the invoice form click on button `Send & Print`.
+
+If the invoice matches the configured domain for PEPPOL, the invoice will be generated
+and validated according to the stricter PEPPOL 3.0 BIS standard.
```

Comment:
For convenience, please add a link to the validator. And add as instruction to validate the generated file.
While reviewing this module I noticed that in the current workflow a couple of parameters needs to be configured to generate a correct peppol invoice.
(In current workflow there are no warnings if misconfigured something which results that your generated file is not compliant to the peppol standard.)